src/solvers/td3.ml

```diff
@@ -35,6 +35,7 @@ module WP =
       mutable side_infl: VS.t HM.t; (** Influences to side-effected variables. Not normally in [infl], but used for restarting them. *)
       mutable var_messages: Message.t HM.t;
       mutable rho_write: S.Dom.t HM.t HM.t;
+      mutable dep: VS.t HM.t
```

Comment:
Should do the proposed `dep` → `uses` rename then and add documentation to this field.

Reply:
The way it is implemented now, it is actually the `dep` from the TD3 paper though

Reply:
Oh, right, yes!